_rules/html-page-lang-valid-bf051a.md

```diff
@@ -34,7 +34,7 @@ For each test target, the `lang` attribute has a [valid language subtag](#valid-
 
 ## Assumptions
 
-This rule assumes that the presence of a `lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.
+This rule assumes that the presence of a `lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG. WCAG only requires that the language of the page can be "programmatically determined", but this could be achieved by using other means than a `lang` attribute (e.g., HTTP headers, `meta` element, …). Thus, an invalid `lang` attribute could be overridden by some other programmatic technique and not cause any accessibility issue or WCAG failure; but this rule would nonetheless fail.
```

Comment:
How about:

```suggestion
There are other methods in HTML by which the language of the page is set; using http-headers or using the `meta` element. These are generally not supported by assistive technologies. This rule assumes that these two methods are insufficient to satisfying [success criterion 3.1.1 language of the page](https://www.w3.org/TR/WCAG21/#language-of-page).
```

Reply:
I like it. Rephrased a bit to be less restrictive (there may be ways we haven't think of, these are quite likely not well supported by AT…)